assignment2.java

```diff
@@ -107,7 +115,7 @@ private static String getRTypeInstruction(String instructionName, String binaryI
         } else if (instructionName == "BR") {
```

Comment:
On line 108 do we need to use `binary2ComplimentToInt` for the shamt? I'm not sure if shamt can be negative though.